src/layout/grid/grid.jsx

```diff
@@ -118,6 +118,8 @@ const propTypes = {
      * Deprecated prop. Please use `classes` to override styles.
      */
     verticalAlign: PropTypes.oneOf(['bottom', 'middle', 'top']),
+
+    responsiveQueryType: PropTypes.oneOf(['media', 'container']),
```

Comment:
**NAB** : Would be great, if we can add a single line description indicating what this prop would do?